Brokerages/Bitfinex/BitfinexBrokerage.Messaging.cs

```diff
@@ -589,11 +596,12 @@ private void EmitFillOrder(string[] entries)
                 // if the order is closed, we no longer need it in the active order list
                 lock (closeLocker)
```

Comment:
I think we can safely remove the `lock` statement and the `closeLocker` variable